docs/definitions.md

```diff
@@ -31,6 +30,14 @@ A memory safe by default language prevents (by default) common memory safety vul
 * Mismatched free
 * Unwanted aliasing
 
+A memory safe by default language also addresses the following:
+
+**Race Conditions**
+
+* Concurrent read/writes to shared memory
+
+Some memory safe by default languages prevent data races (such as Rust), but others (such as Java and Go) require the use of additional packages or libraries to prevent data races. Please see [Best Practices for Memory Safe by Default Languages](best-practice-memory-safe-by-default-languages.md)

```

Comment:
I'd say "the use of additional conventions, packages, or libraries to prevent data races".

I think one of the more common approaches for preventing data races is to apply certain conventions. E.g., in Java, using private data and requiring all accesses to be synchronized is one way to prevent data races (though in that case beware of deadlock).

Reply:
That sounds reasonable! Adding.